docs/topics/items.rst

```diff
@@ -15,7 +15,7 @@ especially in a larger project with many spiders.
 
 To define common output data format Scrapy provides the :class:`Item` class.
 :class:`Item` objects are simple containers used to collect the scraped data.
-They provide a `dictionary-like`_ API with a convenient syntax for declaring
+They provide a :class:`dict` like API with a convenient syntax for declaring
```

Comment:
```suggestion
They provide a :class:`dict`-like API with a convenient syntax for declaring
```

I believe the hyphen is important here to avoid the interpretation where API is an example of a `dict`.